docs/src/cism/cism.md

```diff
@@ -327,7 +329,7 @@ h_diff = soln(tₑ).dynamics_h - hₐ
 maximum(abs.(h_diff))
 ```
 
-We compute root meand square error (RMSE) as well, both over the entire domain, and *excluding where the ice distribution is 0 in the analytic solution.* (Considering the entire domain decreases the RMSE, while not telling you much about the dynamics in the area of interest.) Note that the official CISM benchmark reports 6.43 and 9.06 RMSE for their two solver implementations.
+We compute root-mean-square error (RMSE) as well, both over the entire domain, and *excluding where the ice distribution is 0 in the analytic solution.* This is done since considering the entire domain decreases the RMSE while not telling you much about the area of interest. Note that the official CISM benchmark reports `6.43` and `9.06` RMSE for their two solver implementations.
```

Comment:
I liked the sentence better before

Reply:
The use of a parentheses in a sentence is to include an idea which is not essential. I argue that this is explaining why we compute RMSE the way we do and so is definitely essential.